README.md

```diff
@@ -46,7 +42,13 @@ $ npm install
 $ npm run start
 ```
 
-### Developer Tools
+#### Start from [VS Code](https://code.visualstudio.com/)
+
+You need install the [Rax extension](https://marketplace.visualstudio.com/items?itemName=Rax.vscode-rax) and invoke the `rax` command from the VS Code command palette (`Ctrl + Shift + P` or `Cmd + Shift + P` on Mac):
```

Comment:
You need to install [Rax extension](https://marketplace.visualstudio.com/items?itemName=Rax.vscode-rax)...